wntr/epanet/io.py

```diff
@@ -2451,34 +2443,33 @@ def generate_control(self, model):
 
 
 class BinFile(object):
-    """
```

Comment:
This docstring has been giving me fits - there are warnings in every docs build, but I can't find the error. I *think* this has fixed it.